### ISSUE TITLE
Single Player tournament

### ISSUE THREAD

**Issue by aledre** [evroon/bracket#938]
Would like to have possibility to play single player tournament, so register users and choice tournament mode. Now I need to create a team with only one player to do this

**Comment by evroon** [comment]
You can just create teams without any players in it right?

**Comment by aledre** [comment]
Yes, so I can intend team as a player? There are a lot of sports where you can play in team or in single mode (like darts, tennis, tabletennis kicker, ecc). A lot of them are structured in CLUBS with PLAYERS. Sometimes all players can play tournament in single mode to win regional trophy or something else. So I immagine a structure where I have a DB with teams and players for each team and I can create a team tournament of course, ora single (or double) player tournament where I can pick up directly players registered in teams, where every player play for itself (in that tournament, of course)

**Comment by evroon** [comment]
> Yes, so I can intend team as a player?

Yes indeed. I have now added a note about this in the [docs](https://docs.bracketapp.nl/docs/usage/guide#2-register-teams-and-players)

You can just regard the players as teams, so you don't have to add any players to those teams then.
